python/tests/test_fastexcel.py

```diff
@@ -533,6 +533,15 @@ def test_null_values_in_cells() -> None:
     pd_assert_frame_equal(sheet.to_pandas(), pd_expected)
 
 
+def test_invalid_value_num() -> None:
+    excel_reader = fastexcel.read_excel(path_for_fixture("fixture-invalid-cell-value-num.xlsx"))
+    sheet = excel_reader.load_sheet(0)
+
+    expected = {"Column": [8, None]}
```

Comment:
Looks like your column's dtype is `Float64`

```suggestion
    expected = {"Column": [8.0, None]}
```